Marlin/Configuration_adv.h

```diff
@@ -1458,6 +1458,12 @@
     #define SD_FIRMWARE_UPDATE_INACTIVE_VALUE 0xFF
   #endif
 
+  /**
+   * If you have Flash space to spare, Marlin can store your customized
+   * settings and export them to the SD Card as mc.zip with 'M503 C'.
+   */
+  //#define CONFIG_EMBED_AND_SAVE_TO_SD
+
```

Comment:
I think the main point of this feature is to be opt-out (my goal was to be able to rebuild a Marlin from what's installed on the printer, when no source is provided). If you make it opt-in, it'll fail the concept here, no one will enable it.

Reply:
It cannot be opt-out because it increases the build size too significantly (and will lead to a flood of complaints from users who have their options set up to fill the available space). This version of Marlin is still directed to boards with as little as 64KB of flash and still facilitates building with Arduino IDE and we are still counting every byte of the default configuration.

Reply:
Oh. In that case, let's make it opt-out for the rich (those with plenty of flash) and opt-in for the poor. What do you think ?

Reply:
I think, "Oh my god, it feels like I'm the only person going through the issues list and trying to fix the open bugs, so I'd better get back to work! I can't release the next version in this state!"